torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -99,6 +100,28 @@ c10::optional<std::vector<IValue>> runNodeIfInputsAreConstant(
         return c10::nullopt;
       }
     }
+    // see [Constant Object Weak CompilationUnit Reference]
+    if (v.isCustomClass()) {
+      if (v.toObject()->is_weak_compilation_ref()) {
+        continue;
+      }
+      if (!db) {
```

Comment:
@eellison, I have a question about the **db** parameter,  why do we skip  Weak CompilationUnit conversion for **CustomClass** value when **db** is null? in my side, before this PR, when I use **ConstantPropagation** which can make **CustomClass** value(customer_class_output = op(constant inputs)) to  constant, but after this PR, because the default value of **db** is null, so this conversion will be skilled and can't insert  **CustomClass** value, see: https://github.com/pytorch/pytorch/blob/18dd5cdba55bee02d93e391a4103f01a70682940/torch/csrc/jit/ir/constants.cpp#L125-L134

Reply:
@eellison, do you have any point about this? thanks!

Reply:
Hi, sorry, did this PR change the behavior here ? Typically we skipped the Custom Class prop when the aliasdb is null because we don't know whether it will be mutated or not. I don't think this PR changed that behavior... If it did do you mind sending small repro ?